README.md

```diff
@@ -23,6 +23,9 @@ Currently there is only one possible failure (`MissingFieldFailure`) which conta
 It is common that the field has different name in the GPB and in the case class. You can use `GpbName` annotation 
 to override the expected name - see the example.
 
+It can happen the case class has more fields than the GPB and you wan't just ignore them during the conversion. You can use `Ignored`
```

Comment:
_Sometimes the case class has more fields than the GPB and you just want to ignore them during conversion._

_Please **note that** the ignored field in the case class **must** have default value!_